libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -101,6 +106,21 @@ void CodeTransform::operator()(FunctionCall const& _call)
 	solAssert(m_scope, "");
 
 	m_assembly.setSourceLocation(_call.location);
+
+	if (m_builtinFunctions.count(_call.functionName.name))
```

Comment:
Don't see a simple way which would keep it readable. Also note we have two different kinds of builtins:
- simple ones, which fit this scheme
- complex ones, which don't (such as split/combine)

For the second case it might be better to inject a `FunctionDefinition`.